.gitlab-ci.yml

```diff
@@ -71,20 +72,27 @@ test:unit:
 
     # Collect coverage reports
     - mkdir -p tests/unit-coverage && find . -name 'coverage.txt' -exec cp --parents {} ./tests/unit-coverage \;
-    - tar -cvf $CI_PROJECT_DIR/unit-coverage.tar tests/unit-coverage
+    - goveralls
```

Comment:
This should be done in publish:tests instead

Reply:
This is still not resolved. This needs to be done in `publish:tests`. See the standard in other repos.

Reply:
you are too fast! I have had not requested review again yet!

Reply:
is it ok now?

Reply:
> is it ok now?

No, sorry.

We have been doing it as a separate publish job. See [mender](https://github.com/mendersoftware/mender/blob/master/.gitlab-ci.yml#L55) or [mender-artifact](https://github.com/mendersoftware/mender-artifact/blob/master/.gitlab-ci.yml#L143) for example.

The reason is to have a clean way to remove all the codecov ones in one go next week.